includes/definitions/radwin.yaml

```diff
@@ -0,0 +1,11 @@
+os: radwin
```

Comment:
Yeah, from what I can find. When you telnet in to the device, it just spits out a software revision.

Hello admin, welcome to ODU Management CLI!
+-----------------------------------------------------------+
     Software Revision          2.6.00_b2830_May  2 2011
+-----------------------------------------------------------+